src/index.js

```diff
@@ -124,6 +124,18 @@ export default class TimeAgo extends Component<DefaultProps, Props, void> {
     }
   }
 
+  parseDateString (dateString) {
```

Comment:
Also add types.

dateString may not be a string.

Please account for the fact that date may be a Date object, a number of a string.